site/source/docs/migrating_from_3.x.md

```diff
@@ -83,25 +83,144 @@ However, we did introduce some small breaking changes. We removed:
+3. New `dappConnection` magic key word: `$EMBARK`
+    - This tells the Dapp to connect to Embark's proxy and node
+        - Doing so let's you use the same accounts for deployment **and** in the Dapp
+            - Plus, it let's you benefit from a couple of other features inside your Dapp
+    - We recommend putting `$EMBARK` first in the `dappConnection` array in development, as it's the easiest way to develop your Dapp
+    
+
+### Blockchain config
+
+The blockchain configuration changed a lot, but in a good way, as it's **way** simpler now. Hopefully, with all those changes, you'll have an easier time understanding how the different configurations work.
+
+1. Defaults everywhere!
+    - Most of the configs you were used to have in Embark 4 are now put as defaults in Embark
+        - That means you don't have to specify them
+        - You can find the basic configurations [here](/docs/blockchain_configuration.html#Common-Parameters)
+        - If you want to modify some of the advanced parameters, you always can. Just put them in the `clientConfig` object. You can see all the options [here](/docs/blockchain_configuration.html#Advanced-parameters) 
+2. There were a couple of renamed parameters:
+    - `isDev` => `miningMode: 'dev'`
+        - This enables you to control the node's mining mode with only one option, instead of two (`isDev` and `mineWhenNeeded` were both removed)
+    - `mineWhenNeeded` => `miningMode: 'auto'`
+        - As said above, we mixed the two option into `mininMode`
+        - `auto` is the same as `mineWhenNeeded: true`
+    - `ethereumClientName` => `client`
+        - Just a basic rename, because it's shorter and is more agnostic
+3. New `endpoint` parameter
+    - This new parameter is the replacement of the contract config `deployment` section
+    - Before, you had to set `host`, `port` and `type` in the `deployement` section of the contracts config
+    - Now, all the options are in `endpoint`
+        - For example, you can just put `ws://localhost:1234` and Embark will connect to it
+        - If the connection doesn't work, Embark will start the node using the endpoint
+    - This makes it easier to connect to external nodes or change the local node configurations
+4. The blockchain config is now the **only** source of accounts
+    - In Embark 4, there were `account` arrays in both the contract and blockchain configs
+    - In Embark 5, only the blockchain `accounts` remain
+    - The accounts there will be used for contract deployment, node creation **and** in the Dapp
+        - Before you could see the accounts in your Dapp, but couldn't use them
+        - Now, you can fund them, use them and debug with them
+        - This is thanks to our new proxy
+            - All you have to do is set `$EMBARK` as the `dappConnection` in contract config to connect to Embark's proxy in your Dapp
```

Comment:
```suggestion
            - All you have to do is set `$EMBARK` as the `dappConnection` in contracts config to connect to Embark's proxy in your Dapp
```

The dapp file is `config/contracts.js` so I feel it's betters to consistently say *contracts config*. I noticed that here, but I think there are several places the change could be made.